src/components/widgets/hr-payroll-summary/hr-payroll-summary.directive.coffee

```diff
@@ -146,6 +144,10 @@ module.controller('WidgetHrPayrollSummaryCtrl', ($scope, $q, ChartFormatterSvc,
 
   $scope.hasElements = ->
     $scope.selectedElements? && $scope.selectedElements.length > 0
+
+  getIdentifier = (element)->
```

Comment:
I thought about this but I don't think it belongs there, it is not generalised enough in my opinion. It should live within the setting component controller once refactored.